internal/storages/memory/folder.go

```diff
@@ -0,0 +1,83 @@
+package memory
+
+import (
+	"bytes"
+	"github.com/pkg/errors"
+	"github.com/wal-g/wal-g/internal/storages/storage"
+	"io"
+	"io/ioutil"
+	"path/filepath"
+	"strings"
+	"sync"
+)
+
+type Folder struct {
+	path    string
+	Storage *Storage
+}
+
```

Comment:
Let's add here some notes about intended usage. Maybe warn users about possible /dev/null effect of process termination.